sstables/compress.hh

```diff
@@ -367,6 +367,10 @@ public:
     friend class sstable;
 };
 
+// for API query only. Free function just to distinguish it from an accessor in compression
+// accepts nullptr as arg. 
+compressor_ptr get_sstable_compressor(const compression&);
```

Comment:
How does it accept a nullptr when the arg is a reference?

Reply:
Sorry, the comment is stale. First iteration was thusly, but I modified it. Will reword the comment.

Reply:
Removed.